qiskit-superstaq/qiskit_superstaq/superstaq_backend.py

```diff
@@ -136,3 +206,234 @@ def run(
         job = qss.SuperstaQJob(self, job_id)
 
         return job
+
+    def compile(
+        self,
+        circuits: Union[qiskit.QuantumCircuit, List[qiskit.QuantumCircuit]],
+        **kwargs: Any,
+    ) -> qss.compiler_output.CompilerOutput:
+        """Compiles the given circuit(s) to the backend's native gateset.
+
+        Args:
+            circuits: The qiskit QuantumCircuit(s) to compile.
+            kwargs: Other desired compile options.
+
+        Returns:
+            A CompilerOutput object whose .circuit(s) attribute contains optimized compiled
+            circuit(s).
+
+        Raises:
+            ValueError: If `target` is not a valid AQT or IBMQ target.
```

Comment:
nit - something like
```suggestion
            ValueError: If this backend does not support compilation.
```